docs/ibc/middleware/develop.md

```diff
@@ -239,4 +240,21 @@ func SendPacket(appPacket channeltypes.Packet) {
 
     return ics4Keeper.SendPacket(packet)
 }
+
+// middleware must unwrap the channel version so the underlying application 
```

Comment:
nit: godoc should start with function name

Reply:
Yea that makes sense, we should probably fix this for all functions in our docs (was following the existing format)